lib/zizia/hyrax/hyrax_basic_metadata_mapper.rb

```diff
@@ -152,5 +156,10 @@ def basic_fields
          :based_near, :related_url,
          :bibliographic_citation, :source]
       end
+
+      # Properties requires for zizia
+      def zizia_fields
+        [:deduplication_key]
```

Comment:
I'm not sure we can do this for a "Hyrax" mapper because Hyrax out of the box isn't going to have a `deduplication_key`.  Unless we want to create a Zizia generator that adds it to basic metadata ???

Reply:
There are instructions for in the README for adding a **deduplication_key**. If we end up changing the format of the spreadsheets to match Emory's it or something similar will be required.